Bender.yml

```diff
@@ -3,6 +3,7 @@ package:
 
 dependencies:
   fpnew: { git: "https://github.com/pulp-platform/fpnew.git", version: 0.6.1 }
+  common_cells: { git: "https://github.com/pulp-platform/common_cells.git", version: 1.16.4 }
```

Comment:
Why is this needed (the fifo from common_cells was copied; is another cell from that repos now used)?